mappings/net/minecraft/util/math/Box.mapping

```diff
@@ -6,11 +6,17 @@ CLASS net/minecraft/class_238 net/minecraft/util/math/Box
 	FIELD field_1324 maxZ D
 	FIELD field_1325 maxY D
 	METHOD <init> (DDDDDD)V
-		ARG 1 x1
-		ARG 3 y1
-		ARG 5 z1
+		ARG 1 minX
+		ARG 3 minY
+		ARG 5 minZ
+		ARG 7 maxX
+		ARG 9 maxY
+		ARG 11 maxZ
```

Comment:
Same as my other comment, positions can be given in either order so why use min/max.

Reply:
Other occurances exist in BlockPos, vec3d constructors aswell.

Reply:
According to the code you posted above, the `create` methods put the values in the right order for the constructor. Meaning the constructor does care about the order, and the argument names are correct.